.eslintrc

```diff
@@ -5,12 +5,23 @@
     "browser": true,
     "es6": true
   },
-  "parser": "babel-eslint",
-  "ecmaFeatures": {
-    "modules": true,
-    "jsx": true
-  },
   "rules": {
```

Comment:
same here, can we revert this file to include only the single rule. We can perhaps add individual overrides per line.

`no-param-reassign` can be easily fixed by renaming all the reducers from `obj` to `acc` (`acc` is whitelisted as an acceptable param that can be reassigned)